feature-odyssey/src/main/kotlin/com/hedvig/android/odyssey/search/SearchActivity.kt

```diff
@@ -99,6 +108,22 @@ class SearchActivity : ComponentActivity() {
     }
   }
 
+  @Composable
+  private fun Error(message: String, onRetry: () -> Unit) {
+    Column(
+      modifier = Modifier
+        .fillMaxWidth()
+        .padding(16.dp),
+      horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+      Text(text = message)
+      Spacer(modifier = Modifier.padding(top = 8.dp))
+      TextButton(onClick = onRetry) {
+        Text(text = stringResource(id = R.string.GENERAL_RETRY))
+      }
+    }
+  }
+
```

Comment:
Hmm maybe we can try and use what we have in figma here https://www.figma.com/file/tpp00CvD8ALUKdjDRzyygv/Android%E2%80%A8-UI-Kit?node-id=1863-4046 which is this https://github.com/HedvigInsurance/android/blob/76ae326b892b07d1aa42c9d0c9b6b9718231eb8c/core-design-system/src/main/kotlin/com/hedvig/android/core/designsystem/component/information/AppStateInformation.kt#L22-L38 and then you can put a LargeContainedButton under it with a 40dp spacing as that figma file describes.

Maybe we should have a component for this use case btw, right now we do have `GenericInfoScreen` which is exactly that, but it forces the button to be at the bottom of the screen, as that figma shows it should be if it's a full screen error item.